pastepwn/actions/discordaction.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+import logging
+from string import Template
+
+from pastepwn.util import Request, DictWrapper
+from .basicaction import BasicAction
+
+
+class DiscordAction(BasicAction):
+    """Action to send a Discord message to a certain webhook or channel."""
+    name = "DiscordAction"
+
+    def __init__(self, webhook=None, token=None, channel_id=None, custom_payload=None, template=None):
+        super().__init__()
+        self.logger = logging.getLogger(__name__)
+
+        self.webhook = webhook
+        if webhook is None:
+            if token is None or channel_id is None:
+                raise ValueError('Invalid arguments: requires either webhook or token+channel_id arguments')
+            self.token = token
+            self.channel_id = channel_id
+        self.custom_payload = custom_payload
+        if template is not None:
+            self.template = Template(template)
+        else:
+            self.template = None
+
+    def perform(self, paste, analyzer_name=None):
+        """Send a message via Discord to a specified channel, without checking for errors"""
+        r = Request()
+        if self.template is None:
+            text = "New paste matched by analyzer '{0}' - Link: {1}".format(analyzer_name, paste.full_url)
+        else:
+            paste_dict = paste.to_dict()
+            paste_dict["analyzer_name"] = analyzer_name
+            text = self.template.safe_substitute(DictWrapper(paste_dict))
```

Comment:
I created a much better way on how to use templates in the most recent master branch. But no worries, I will fix this later on :)

Reply:
I belive this is Mxrk's work, inspired from `TelegramAction` source code, so make the change there too :)